common/app/views/mainAMP.scala.html

```diff
@@ -40,6 +42,22 @@
 
             @body
 
+            <div class="fc-container__inner">
+                <div class="fc-container__header__title">
+                    <span>popular</span>
+                </div>
+                <amp-list width="600" height="1000" layout="responsive" src="@if(Play.isDev) {https://www.thegulocal.com} else {@Configuration.ajax.url}/most-read-mf2.json">
```

Comment:
re the if statement, I don't know the if statement is adding anything.
It would need https enabled in nginx to work, and if someone has set that up, they might as well access the site using the https url anyway.  It would be more amp compliant to do that anyway.  Or is it better to use http where possible in dev mode?